phpunit/directives/wp-process-directives.php

```diff
@@ -47,4 +47,18 @@ function( $p ) {
 		$tags   = new WP_HTML_Tag_Processor( $markup );
 		wp_process_directives( $tags, 'foo-', array(), $attribute_directives );
 	}
+
+	public function test_directives_with_type_and_name_are_processed_correctly() {
```

Comment:
I called `type` and `name` the parts before and after the colon, respectively. Let me know if you come up with a better naming!

Reply:
I can't right now 🤔 I think it's fine to merge as-is, we can change later.

We could try avoiding naming things altogether by calling the test something like

```suggestion
	public function test_directives_with_colon_processed_correctly() {
```

😬